src/pipecat/services/assemblyai/stt.py

```diff
@@ -258,11 +262,13 @@ async def _receive_task_handler(self):
                 except websockets.exceptions.ConnectionClosedOK:
                     break
                 except Exception as e:
-                    logger.error(f"Error processing WebSocket message: {e}")
+                    logger.error(f"{self} exception: {e}")
+                    await self.push_error(ErrorFrame(error=f"{self} error: {e}", fatal=True))
```

Comment:
Same.Don't make it `fatal`. I believe we should just remove `fatal=X` form this PR and use the default (which is `False`). `Fatal` means the whole bot should end.